webservice/app/Providers/EventServiceProvider.php

```diff
@@ -26,20 +26,5 @@ class EventServiceProvider extends ServiceProvider
     public function boot()
     {
         parent::boot();
-
-        Event::listen('tymon.jwt.absent', function () {
```

Comment:
Yes, in this release these events and the middleware that were in Kernel.php aren't used.